hack/test-update-storage-objects.sh

```diff
@@ -112,8 +112,8 @@ tests=(
 examples/persistent-volume-provisioning/rbd/rbd-storage-class.yaml,storageclasses,,slow,v1beta1,v1
 )
 
-KUBE_OLD_API_VERSION="storage.k8s.io/v1beta1,extensions/v1beta1"
-KUBE_NEW_API_VERSION="storage.k8s.io/v1,extensions/v1beta1"
+KUBE_OLD_API_VERSION="networking.k8s.io/v1,storage.k8s.io/v1beta1,extensions/v1beta1"
+KUBE_NEW_API_VERSION="networking.k8s.io/v1,storage.k8s.io/v1,extensions/v1beta1"
```

Comment:
It looks like we should disable `extensions/v1beta1` in the new api version, because we want to make sure we can read from `networking.k8s.io/v1` even if `extensions/v1beta1` is removed.

At line 111, we should add a test data for networking.k8s.io object.

@sttts you only changed the internal versions of the PR, so it's not your responsibility to add the upgrade test. It should had been added when we created networking.k8s.io/v1.